codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/GoIntegration.java

```diff
@@ -152,6 +152,17 @@ default void addConfigInterfaceFields(
         // pass
     }
 
+    /**
+     * Finalizes the model. This plugin can be used to add RuntimeClientPlugins
```

Comment:
This doesn't finalize the model. It just lets you look at the finalized model before codegen happens / before runtime plugins are added.